### PR TITLE
#304 Added function to sort drop down menus in place using Javascript

### PR DESCRIPTION
**Pull request by angarg12**
Added a new function to admUtils to sort drop down menus.
Modified main.xhtml so that this function is triggered site wide.

---
This change is [Reviewable](https://reviewable.io/reviews/servicecatalog/development/314)

### CODE REVIEW

**Comment by DawidCh** [comment]
Ok. Here is one of my commits:
https://github.com/servicecatalog/development/pull/310/commits/89bd6462bcc1e5665908eccf650278efde0f8cc8
Do you see what has been modified? Red lines has been removed and green added. In your commit I see that every line has been modified. Did you modify every line? I do not think so. Do not use "format all lines" from eclipse and do not use "convert to unix" because it modifies whole file and your changes are gone in the crowd.

**Comment by angarg12** [comment]
Did you even check the new pull request?

**Comment by GoebelL** [comment]
Hi,
@MiethanerR has just added a [description of the formatting rules](https://github.com/servicecatalog/development#setting-up-a-workspace) in the OSCM development Readme. 


@DawidCh 
Our contributors will be grateful if you add the information regarding the settings "format all lines" and "convert to unix".

Thanks.

**Comment by DawidCh** [comment]
@angarg12 Yes I have checked. This is new pull request. The old one is #312. The new one (this) pull request contains branch fb_github_304 from repository angarg12. The branch is 2 commits ahead and 836 commits behind servicecatalog:master. Commits which are ahead:
https://github.com/angarg12/development/commit/c9c8cc73ace59a5b7570291a9d679790a7caacd1
with  2,444 additions and 2,440 deletions
https://github.com/angarg12/development/commit/6cd76b840f9f07e5a54b347748ac685513f45a72
with 2,441 additions and 2,408 deletions
What from I see pull request #312 and #314 (this one) are the same. They contained the same commits, with the same modification of two whole files.
If I am wrong about the situation, just put more light on it, please.
Tip: do not format or modify line endings in the files, please.

**Comment by angarg12** [comment]
@DawidCh my first commit modified all files because the IDE was changing the line endings.

I did the second commit to fix that, therefore changing the whole files again.

If you take both commits together the changes are +42 −5.

**Comment by DawidCh** [comment]
Ok, fine. Could you please rebase your branch on master (to resolve conflicts) and create pull request to branch https://github.com/servicecatalog/development/tree/fb_github_304 ? We need to test it first before merging it to master.
Thanks.

File: oscm-portal/WebContent/marketplace/templates/main.xhtml [hunk 1]
```diff
@@ -188,6 +192,7 @@
     AdmUtils.contentsTitle();
     AdmUtils.setInfoPanel("progressMessages");
     AdmUtils.setInfoPanel("infoMessages");
+<<<<<<< HEAD
```
Comment (DawidCh) [inline oscm-portal/WebContent/marketplace/templates/main.xhtml]: Thanks for modification of pull request. I see you have rebased/merged master to branch. Are you sure that lines 195, 210, 212 should be contained in the file?

File: oscm-portal/WebContent/scripts/admUtils.js [hunk 2]
```diff
@@ -2213,6 +2213,7 @@ AdmUtils.adjustDialogHeightOnResize = function(dialogId) {
 	$("#" + dialogId + "_content").height(dialogHeight + 100);
 }
 
+<<<<<<< HEAD
```
Comment (DawidCh) [inline oscm-portal/WebContent/scripts/admUtils.js]: The same here, 2268 and 2300?

File: oscm-portal/WebContent/marketplace/templates/main.xhtml (hunk 1 above)
Comment (angarg12) [inline oscm-portal/WebContent/marketplace/templates/main.xhtml]: Check now please

**Comment by DawidCh** [comment]
Reviewed 2 of 2 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/servicecatalog/development/314)*

**Comment by DawidCh** [comment]
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[oscm-portal/WebContent/marketplace/templates/main.xhtml, line 195 at r2](https://reviewable.io:443/reviews/servicecatalog/development/314#-KRn6S1_rkc5sPVd2mJZ-r2-195:-KSFOj_-8vy7D0N1lOAl:b32w0x) ([raw file](https://github.com/servicecatalog/development/blob/98620eec9fd67e22f425e307bb1c179f36a1fdec/oscm-portal/WebContent/marketplace/templates/main.xhtml#L195)):*
<details><summary><i>Previously, angarg12 wrote…</i></summary>
> Check now please
</details>
OK

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/servicecatalog/development/314)*

**Comment by DawidCh** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[oscm-portal/WebContent/scripts/admUtils.js, line 2216 at r2](https://reviewable.io:443/reviews/servicecatalog/development/314#-KRn6S1_rkc5sPVd2mJ_-r2-2216:-KSFOxF8-E65hgHqngkt:b32w0x) ([raw file](https://github.com/servicecatalog/development/blob/98620eec9fd67e22f425e307bb1c179f36a1fdec/oscm-portal/WebContent/scripts/admUtils.js#L2216)):*
<details><summary><i>Previously, DawidCh (Dawid) wrote…</i></summary>
> The same here, 2268 and 2300?
</details>
OK

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/servicecatalog/development/314)*

**Pull request merged**